modules/installation-gcp-permissions.adoc

```diff
@@ -35,6 +35,14 @@ When you attach the `Owner` role to the service account that you create, you gra
 .Required roles for using the Cloud Credential Operator in passthrough mode
 * Compute Load Balancer Admin
 
+.Required roles for using short-lived credentials
+* IAM Service Account User
+
+[NOTE]
+====
+If the created service account does not include a key, or you are authenticating via short-lived credentials or Default Application credentials, you must include the roles listed in Required roles for using short-lived credentials.
+====
```

Comment:
It looks like adding a note in the middle of the list of permissions is breaking something in asciidoc. Can you move this note down a few lines to be right after the `endif::template[]` line? I think you'll need an empty new line before and after the note block.